lib/pbench/server/api/resources/users_api.py

```diff
@@ -282,9 +282,9 @@ def post(self):
                 self.logger.exception("Exception occurred deleting an auth token")
                 abort(500, message="INTERNAL ERROR")
             else:
-                self.logger.debug("User {} logged out", user.username)
+                self.logger.warning("User {} logged out", user.username)
         else:
-            self.logger.debug("User logout with invalid or expired token")
+            self.logger.warning("User logout with invalid or expired token")
```

Comment:
Tricky. A logout with an "invalid" token is clearly a user error and therefore `warning`. But is logging out with an expired token really a "user error" or just an unusual event possibly worth noting (info level)?? Hmm...

Reply:
Seems like this would be best kept as an informational.

Reply:
> is logging out with an expired token really a "user error" or just an unusual event possibly worth noting (info level)??

Logging out with an expired token shouldn't actually be an error, and it might not even be unusual, depending on how quickly tokens expire and how the Dashboard maintains them.  I can easily imagine someone staring at graphs for an hour and then logging out...which, I think, absent automatic maintenance, would use an expired token.